### PR TITLE
Detach

### PR DESCRIPTION
**Pull request by yanivefraim**

### CODE REVIEW

File: generators/generator-yoshi-fullstack/generators/app/templates/src/components/App/App.spec.js [hunk 1]
```diff
@@ -6,8 +6,13 @@ import App from './App';
 
 describe('App', () => {
   it('renders a title correctly', () => {
-    const wrapper = mount(
-      <App/>
+    let wrapper;
+
+    afterEach(() => wrapper.detach());
```
Comment (ronami) [inline generators/generator-yoshi-fullstack/generators/app/templates/src/components/App/App.spec.js]: move `afterEach()` outside of `it()`

File: generators/generator-yoshi-web/generators/react/templates/src/components/App/App.spec.js [hunk 2]
```diff
@@ -6,8 +6,13 @@ import App from './App';
 
 describe('App', () => {
   it('renders a title correctly', () => {
-    const wrapper = mount(
+    let wrapper;
+
+    afterEach(() => wrapper.detach());
```
Comment (ronami) [inline generators/generator-yoshi-web/generators/react/templates/src/components/App/App.spec.js]: move `afterEach()` outside of `it()`

File: generators/generator-yoshi-fullstack/generators/app/templates/src/components/App/App.spec.js (hunk 1 above)
Comment (yanivefraim) [inline generators/generator-yoshi-fullstack/generators/app/templates/src/components/App/App.spec.js]: מביך...

File: generators/generator-yoshi-web/generators/react/templates/src/components/App/App.spec.js (hunk 2 above)
Comment (yanivefraim) [inline generators/generator-yoshi-web/generators/react/templates/src/components/App/App.spec.js]: מביך...

File: examples/fullstack-react/src/components/App/App.spec.js [hunk 3]
```diff
@@ -5,10 +5,15 @@ import {mount} from 'enzyme';
...
+
+  afterEach(() => wrapper.detach());
+
   it('renders a title correctly', () => {
-    const wrapper = mount(
-      <App/>
-    );
+    wrapper = mount(
+      <App/>,
+      {attachTo: document.createElement('div')});
```
Comment (ronami) [inline examples/fullstack-react/src/components/App/App.spec.js]: bad identation :(

File: generators/generator-yoshi-web/generators/react/templates/src/components/App/App.spec.js [hunk 4]
```diff
@@ -5,9 +5,14 @@ import {mount} from 'enzyme';
 import App from './App';
 
 describe('App', () => {
+  let wrapper;
+
+  afterEach(() => wrapper.detach());
+
   it('renders a title correctly', () => {
-    const wrapper = mount(
+    wrapper = mount(
```
Comment (ronami) [inline generators/generator-yoshi-web/generators/react/templates/src/components/App/App.spec.js]: can probably turn into one line like previous
